pkg/image/definition.go

```diff
@@ -13,8 +13,10 @@ import (
 )
 
 const (
-	TypeISO = "iso"
-	TypeRAW = "raw"
+	TypeISO           = "iso"
+	TypeRAW           = "raw"
+	TypeTar           = "tar"
+	TypeCombustionIso = "combustion-iso"
```

Comment:
These should be named `CombustionTypeTar` and `CombustionTypeISO` in my opinion. The value for the latter should also just be `iso`.